contracts/governance/Governor.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+/**
+ * @title Governor
+ * @dev The Governor holds the rights to stage and execute contract calls i.e. changing Livepeer protocol parameters. 
+ */
+contract Governor {
+    
+    using SafeMath for uint256;
+
+    address public owner;
+
+    /// @dev mapping of updateHash (keccak256(update) => executeBlock (block.number + delay)
+    mapping(bytes32 => uint256) updates; 
+
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+    event UpdateStaged(Update update, uint256 delay);
+
+    event UpdateExecuted(Update update);
+
+    event UpdateCancelled(Update update);
+
+    struct Update {
+        address[] target;
+        uint256[] value;
+        bytes[] data;
+    }
+
+   /// @notice Throws if called by any account other than the owner.
+    modifier onlyOwner() {
+        require(msg.sender == owner, "unauthorized");
+        _;
+    }
+
+
+    /// @notice Throws if called by any account other than this contract.
+    /// @dev Forces the `stage/execute` path to be used to call functions with this modifier instead of directly.
+    modifier onlyThis() {
+        require(msg.sender == address(this), "unauthorized");
```

Comment:
Should the revert reason be more specific to differentiate between other authorization related modifiers (i.e. `onlyOwner`)? i.e. "unauthorized: msg.sender not Governor"